spec/features/ctc/javascript_warnings_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+
+RSpec.feature "during CTC intake clients see a warning on pages that require JS when they have it disabled" do
+  before do
+    allow_any_instance_of(Routes::CtcDomain).to receive(:matches?).and_return(true)
+  end
+
+  scenario "client visits js required pages with javascript turned on", js: true do
+    visit "/en/questions/legal-consent"
+    expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.legal_consent.title'))
+    expect(page).not_to have_text "Please enable Javascript in your browser or use a different browser."
+
+    create_client_and_sign_in
+
+    visit "/en/questions/spouse-info"
+    expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.spouse_info.title'))
+    expect(page).not_to have_text "Please enable Javascript in your browser or use a different browser."
+
+    # visit "/en/questions/confirm-legal"
+    # expect(page).to have_selector("h1", text: I18n.t("views.ctc.questions.confirm_legal.title"))
+    # expect(page).not_to have_text "Please enable Javascript in your browser or use a different browser."
```

Comment:
the story says to do this on the confirm-legal page, but i don't see a script tag on that page. maybe there's somewhere else i should look...?